paasta_tools/setup_kubernetes_cr.py

```diff
@@ -281,10 +277,9 @@ def format_custom_resource(
         "spec": instance_config,
     }
 
-    url = get_dashboard_url(kind, service, instance, cluster)
+    url = get_dashboard_base_url(kind, cluster)
     if url:
-        resource["metadata"]["annotations"]["yelp.com/dashboard_url"] = url
-        resource["metadata"]["annotations"][paasta_prefixed("dashboard_url")] = url
+        resource["metadata"]["annotations"][paasta_prefixed("dashboard_base_url")] = url
```

Comment:
why don't we prefix this one for flink like `dashboard_url` if we are the only ones using it?

Reply:
My idea is that any label or annotation that comes from PaaSTA should be prefixed with PaaSTA as opposed to Flink. For the annotation operator is adding it should have Flink prefix because that annotation is added by operator not PaaSTA.

Reply:
I believe that the guidance that @Rob-Johnson set is to group them by project, not by source: https://yelpwiki.yelpcorp.com/display/COMPUTE/Operator+development+guidelines#Operatordevelopmentguidelines-Labelsandannotations

Reply:
I thought this was limited to labels and annotations attached by operator. Why I would prefer it to be PaaSTA prefixed is because this annotation is added by PaaSTA, if we face any problem we can directly point to the source (in this case PaaSTA) which adds those annotations/labels. Also, how I look at PaaSTA annotations/labels is that operators will not update them, if operator needs to make an update they can add their own annotation with modification.

I am happy to update it if you think the reasoning is not justified.

Reply:
It's reasonable and I can live with it. Let's move ahead.